fe/fe-core/src/main/java/com/starrocks/analysis/IndexDef.java

```diff
@@ -139,6 +140,24 @@ public void checkColumn(Column column, KeysType keysType) throws AnalysisExcepti
         }
     }
 
+    public void verifyColumn(Column column, KeysType keysType) {
```

Comment:
This is because the new framework does not support AnalysisException this function use to remove throw AnalysisException define.I will mark @Deprecated to old method.